cfme/tests/v2v/test_v2v_migrations_ui.py

```diff
@@ -370,3 +370,26 @@ def test_migration_rbac(appliance, new_credential, v2v_providers):
         # Checks migration option is enabled in navigation
         assert 'Migration' in nav_tree['Compute'], ('Migration not found in nav tree, '
                                                     'rbac should allow this')
+
+
+@pytest.mark.parametrize('form_data_single_datastore', [['nfs', 'nfs']], indirect=True)
+def test_edit_mapping_description(appliance, v2v_providers, form_data_single_datastore,
+                                    host_creds, conversion_tags, soft_assert):
+    infrastructure_mapping_collection = appliance.collections.v2v_mappings
+    mapping = infrastructure_mapping_collection.create(form_data_single_datastore)
+    edited_name = 'edited_name_{}'.format(fauxfactory.gen_alphanumeric())
+    edited_form_data = {
+        'general': {
+            'name': edited_name,
+            'description': "my edited description"},
+        'cluster': {},
+        'datastore': {},
+        'network': {}
+    }
+    mapping.update(edited_form_data)
+
+    view = navigate_to(infrastructure_mapping_collection, 'All')
+    if appliance.version >= '5.10':
```

Comment:
Why is this only for `>5.10` ??

Reply:
In additions to @kedark3, there are no checks for < 5.10

Reply:
Not applicable for version 5.9 . Uncollected.